ramls/openTransactions.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "User Transactions Schema",
+  "description": "Schema describing open transactions of a user",
+  "type": "object",
+  "properties": {
+    "userId": {
+      "type": "string",
+      "description": "User's id"
+    },
+    "userBarcode": {
+      "type": "string",
+      "description": "User's barcode"
+    },
+    "hasOpenTransactions": {
+      "type": "boolean",
+      "description": "Specifies if the user has any open transactions"
```

```suggestion
      "description": "Specifies if the user has any open transactions (loans, requests, fees/fines, proxies, blocks)"
```
Explain what "transactions" means.